pandas/core/arrays/categorical.py

```diff
@@ -1505,6 +1505,9 @@ def get_values(self):
             A numpy array of the same dtype as categorical.categories.dtype or
             Index if datetime / periods.
         """
+        raise Exception("USING GET_VALUES")
```

Comment:
Should I be concerned that CI is passing with this here? :)

Perhaps we need a test ensuring that `Categorical.get_values()` raises a warning.

Reply:
> Should I be concerned that CI is passing with this here? :)

No, that's a good sign :) 
As that was my way to find all the places where it was being called, to fix those (although it is not a perfect way, as in certain places the error could also be catched)

But for sure still need to convert this into a warning and add tests that checks that.